RoofingCompanyInserts.sql

```diff
@@ -650,7 +650,7 @@ insert into PlannedProductionEmployeeDetails (IdProces, IdEmployee, StartDate, E
 	(9,4, '2019-08-15', '2019-09-25'),
 	(10,1, '2019-08-15','2019-09-30');
 
- insert into ProductionProces(IdPlan, StartDate, EndDate)
+ insert into ProductionProcess(IdPlan, StartDate, EndDate)
```

Comment:
ta tabela zmienia nazwę co 5 minut- proszę przekazac grupie że prawidlowa nazwa to `ProductionProcess` i taka ma zostac